strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/config/StorageApiConfig.java

```diff
@@ -33,30 +33,30 @@
     @Inject
     private List<ArtifactCoordinatesValidator> versionValidators;
 
-    @Bean(name = "checksumCacheManager")
-    ChecksumCacheManager checksumCacheManager()
+    @Bean(name = "calculatedChecksumCacheManager")
+    protected CalculatedChecksumCacheManager calculatedChecksumCacheManager()
     {
-        ChecksumCacheManager checksumCacheManager = new ChecksumCacheManager();
-        checksumCacheManager.setCachedChecksumExpiredCheckInterval(300000);
-        checksumCacheManager.setCachedChecksumLifetime(60000);
+        CalculatedChecksumCacheManager calculatedChecksumCacheManager = new CalculatedChecksumCacheManager();
+        calculatedChecksumCacheManager.setCachedChecksumExpiredCheckInterval(300000);
+        calculatedChecksumCacheManager.setCachedChecksumLifetime(60000);
 
-        return checksumCacheManager;
+        return calculatedChecksumCacheManager;
     }
 
     @Bean(name = "versionValidators")
-    LinkedHashSet<ArtifactCoordinatesValidator> versionValidators()
+    protected LinkedHashSet<ArtifactCoordinatesValidator> versionValidators()
```

Comment:
Out of curiousity, why are these all `protected`?

Reply:
`Codacy` reported a warning to explicitly provide visibility - in my opinion, we could shut it out, is possible, @steve-todorov

Reply:
> Codacy reported a warning to explicitly provide visibility
I disabled it in codacy.. hopefully it's not reporting it anymore :)